execute_script.bat converted for other OS-es

compute-e opened the pull request:
@JoshSCF hello.
This PR is for my work to convert the file execute_script.bat,  
for (Microsoft) Windows, to be used on other operating system(OS)s.

It **_is not ready for you to merge before_** I add a comment saying that it is.

When you merge this pull request, can you please do the [Merge pull request option](https://help.github.com/articles/merging-a-pull-request/#merging-a-pull-request-on-github)  
(or click the merge drop down menu and select _Create a merge commit_) to  
_Merge all of the commits into the base branch_, not do a weird single commit,  
so my repository fork shows the (head) branch as up-to-date, done that merge.

user-e committed:
Update README.md for better formatting, small edits

user-e committed:
Use relative filename and a batch code comment.

In one of the six forks of your repository I saw a relative filename used
for the Python file - store it in the current folder with the batch code file.

user-e committed:
Merge branch 'master' into original-JoshScript

user-e committed:
Sorted out wording, added new details in README.md

user-e committed:
Use small headings, add my things in README.md

user-e committed:
Fix small problems, adding my things in README.md

user-e committed:
An about section started after problems fixed in README.md

user-e committed:
Merge pull request #2 from JoshSCF/master

Cubed added and Joshscript version changed

user-e committed:
Update association.bat

user-e committed:
Update README.md

user-e committed:
Update README.md

user-e committed:
Update README.md

user-e committed:
Merge branch 'master-version' into original-JoshScript

user-e committed:
Merge pull request #3 from user-e/original-JoshScript

Changes in original-JoshScript

user-e committed:
Update README.md

user-e committed:
Rename association.bat to execute_script.bat

user-e committed:
Update README.md to undo previous commit

I should've used a duplicate of the "original..." branch for the pull
(and previous "conflict" commit) then maybe deleted it after the merge.

user-e committed:
Update README.md

user-e committed:
Update README.md

user-e committed:
Merge pull request #4 from user-e/original-JoshScript

Changes in original-JoshScript

user-e committed:
Merge branch 'master-version' into original-to-master-merges

user-e committed:
Update README.md

user-e committed:
Merge pull request #5 from user-e/original-to-master-merges

Original to master merge (changes in original-JoshScript)

user-e committed:
Update README.md

user-e committed:
Update README.md

user-e committed:
Update README.md

user-e committed:
Update execute_script.bat

user-e committed:
Update execute_script.bat

user-e committed:
Merge pull request #7 from user-e/original-JoshScript

Update execute_script.bat - changes in original-JoshScript

user-e committed:
Merge pull request #8 from user-e/master-version

Copy master-version branch to original-to-master-merges

user-e committed:
Version number M.m.p (m=add function, p=bug fixes)

<https://help.github.com/articles/creating-releases/>, <https://semver.org/#semantic-versioning-200>
("Semantic Versioning 2.0.0 Summary"):
Given a version number MAJOR.MINOR.PATCH, increment the:
 - MAJOR version when you make incompatible API changes,
 - MINOR version when you add functionality in a backwards-compatible manner, and
 - PATCH version when you make backwards-compatible bug fixes.
Additional labels for pre-release and build metadata are available
as extensions to the MAJOR.MINOR.PATCH format.

user-e committed:
Version number M.m.p (m=add function, p=bug fixes)

https://github.com/user-e/E-Script/commit/42b78e8da3c71d375c486fe789c28399fc10c21b

user-e committed:
Version number M.m.p (m=add function, p=bug fixes)

https://github.com/user-e/E-Script/commit/42b78e8da3c71d375c486fe789c28399fc10c21b

user-e committed:
Version number M.m[.p] (m=add function, p=bug fixes)

user-e committed:
Version number M.m[.p] (m=add function, p=bug fixes)

user-e committed:
Application title bar has full version no. (M.m.p)

user-e committed:
Merge branch 'original-to-master-merges' into temporary

user-e committed:
Merge pull request #11 from user-e/temporary

Changes in original-JoshScript

user-e committed:
Update execute_script.bat

user-e committed:
Update execute_script.bat

user-e committed:
Merge pull request #13 from JoshSCF/master

Some edits of README and .bat file, update to v1.3.0 (#7)

user-e committed:
Merge pull request #14 from user-e/original-JoshScript

Merge changes from original-JoshScript

user-e committed:
Update README.md

user-e committed:
Update README.md

user-e committed:
Create exec_script.linux.sh

Work in progress, started it.

user-e committed:
Update exec_script.linux.sh

Unfinished work in progress continued.

user-e committed:
Use official OS names, from Wikipedia articles.

user-e committed:
Update README.md p1 /2, add exec_script file info.

user-e committed:
Update README.md part 2 /2, add more info.

user-e committed:
Update execute_script.bat

user-e committed:
Update exec_script.linux.sh

user-e committed:
Update README.md

user-e committed:
Update exec_script.linux.sh

user-e committed:
Update exec_script.linux.sh

user-e committed:
Code needed quotes, in exec_script.linux.sh

user-e committed:
Create exec_script.mac.command

user-e commented:
> When you merge this pull request, can you please do the [Merge pull request option](https://help.github.com/articles/merging-a-pull-request/#merging-a-pull-request-on-github)
> (or click the merge drop down menu and select _Create a merge commit_) to
> _Merge all of the commits into the base branch_, not do a weird single commit,
> so my repository fork shows the (head) branch as up-to-date, done that merge.

Read the quote above.

user-e commented:
> When you merge this pull request, can you please do the [Merge pull request option](https://help.github.com/articles/merging-a-pull-request/#merging-a-pull-request-on-github)
> (or click the merge drop down menu and select _Create a merge commit_) to
> _Merge all of the commits into the base branch_, not do a weird single commit,
> so my repository fork shows the (head) branch as up-to-date, done that merge.

@JoshSCF this Pull Request is ready for you to merge.
Can you read my comments, review changes and
merge this pull request _**using the quote above**_, please.

user-e committed:
Merge branch 'original-to-master-merges' into merge-temp

user-e committed:
Update README.md

user-e committed:
Merge branch 'merge-temp' into original-JoshScript

user-e committed:
Update README.md

Reverted https://github.com/user-e/E-Script/commit/8681f9f18f329453e81a5a8fd17e2033ba0f5176.
Avoid doing this any more times.

user-e committed:
Update README.md

compute-e reviewed (commented):
Fixed problems - this Pull Request is now ready for you to merge.
_Read my previous comment and merge this pull request using it, please._

user-e committed:
Update README.md to start about section.

user-e committed:
Update README.md
